tests/lax_autodiff_test.py

```diff
@@ -1055,18 +1055,18 @@ def f2(x, y):
 
   # TODO(mattjj): make this a more systematic test
   def testRemainder(self):
-    rng = np.random.RandomState(0)
+    rng = self.rng()
     x = rng.uniform(-0.9, 9, size=(3, 4))
     y = rng.uniform(0.7, 1.9, size=(3, 1))
     assert not set(np.unique(x)) & set(np.unique(y))
-    tol = 1e-1 if jtu.num_float_bits(np.float64) == 32 else 1e-3
+    tol = 1e-1
```

Comment:
This seems like a pretty loose tolerance for a 64-bit computation.

Reply:
The test fails with the old tolerance.

Options:
- continue hard-coding the random seed
- keep this adjustment
- disable the test

Which do you think is best?

Reply:
Bumping the tolerance and leaving a TODO seems fine.